exporter/humioexporter/config.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2021, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package humioexporter
+
+import (
+	"errors"
+	"net/url"
+	"path"
+
+	"go.opentelemetry.io/collector/config/confighttp"
+	"go.opentelemetry.io/collector/config/configmodels"
+	"go.opentelemetry.io/collector/exporter/exporterhelper"
+)
+
+const (
+	basePath         = "api/v1/ingest/"
+	unstructuredPath = basePath + "humio-unstructured"
+	structuredPath   = basePath + "humio-structured"
+)
+
+// Humio configuration settings specific to logs
+type LogsConfig struct {
+	// The name of a custom log parser to use, if no parser is associated with the ingest token
+	LogParser string `mapstructure:"log_parser"`
+}
+
+// Humio configuration settings specific to traces
```

Comment:
Did you get a linting failure here? Go comments for public types/funcs should start with their type/func name. Like:

```suggestion
// TracesConfig represents the Humio configuration settings specific to traces
```

Reply:
I did not get any linting errors (I have gotten them for typos elsewhere), but I will make sure to follow this convention throughout!